utils/message_html_to_component.jsx

```diff
@@ -60,6 +60,11 @@ export function messageHtmlToComponent(html, isRHS, options = {}) {
                     <LinkTooltip
                         href={node.attribs[hrefAttrib]}
                         title={children[0]}
+                        dataAttributes={{
+                            'data-hashtag': attribs['data-hashtag'],
+                            'data-link': attribs['data-link'],
+                            'data-channel-mention': attribs['data-channel-mention'],
```

Comment:
@mickmister Heads up that `eslint` complains about these lines:
```
/root/mattermost-webapp/utils/message_html_to_component.jsx
 
  64:45  error  'attribs' is not defined  no-undef
 
  65:42  error  'attribs' is not defined  no-undef
 
  66:53  error  'attribs' is not defined  no-undef
 ```

Reply:
Thank you, fixed now